osu.Game/Screens/Menu/IntroWelcome.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osuTK;
+using osu.Framework.Allocation;
+using osu.Framework.Audio;
+using osu.Framework.Audio.Sample;
+using osu.Framework.Screens;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.Textures;
+using osuTK.Graphics;
+
+namespace osu.Game.Screens.Menu
+{
+    public class IntroWelcome : IntroScreen
+    {
+        protected override string BeatmapHash => "64e00d7022195959bfa3109d09c2e2276c8f12f486b91fcf6175583e973b48f2";
+        protected override string BeatmapFile => "welcome.osz";
+        private const double delay_step_two = 2142;
+        private SampleChannel welcome;
+        private SampleChannel pianoReverb;
+
+        [BackgroundDependencyLoader]
+        private void load(AudioManager audio)
+        {
+            Seeya = audio.Samples.Get(@"Intro/welcome/seeya");
```

Comment:
unnecessarily loading two samples (the base is also loading). revert the `SampleChannel` to private and make a `virtual string SeeyaSampleName => "Intro/seeya"`